draft-ietf-tls-tlsflags.xml

```diff
@@ -150,6 +150,13 @@
             <t> A server MAY indicate support for flag-type features in a flags extension of NewSessionTicket (NST).
                 This message has no client-side response.</t>
             <t> In summary, unsolicited flags may appear only in ClientHello, CertificateRequest, and NewSessionTicket.</t>
+            <t> An implementation that receives an invalid tls_flags extension MUST terminate the TLS handshake with a
+                fatal illegal_parameter alert. Such invalid tls_flags extensions include: <list style="symbols">
+                <t> A zero-length extension</t>
+                <t> Multiple tls_flags extensions for the same message </t>
+                <t> A flag set in the tls_flags extension of the wrong message, as specified in the document for that flag </t>
```

Comment:
This seems unimplementable. If I receive a message with a flag I don't recognize, I have no idea whether the flag is allowed in the message or not. Yet this text says I MUST enforce this rule. (There's probably some corresponding wording in RFC8446 we can borrow.)

Nit: It also seems better to phrase this in terms of the registry, rather than the document. We might have multiple documents for the flag if we have to update it.